zerver/webhooks/stripe/view.py

```diff
@@ -161,14 +170,45 @@ def default_body(update_blacklist: List[str]=[]) -> str:
     if category == 'invoiceitem':  # nocoverage
         body = default_body(update_blacklist=['description'])
         if event == 'created':
-            body += ' for {amount}'.format(amount=amount_string(object_['amount'], object_['currency']))
+            body += ' for {amount}'.format(amount=amount_string(
+                object_['amount'], object_['currency']))
     if category.startswith('issuing'):  # nocoverage
         # Not implemented
         raise NotImplementedEventType()
     if category.startswith('order'):  # nocoverage
         # Not implemented
         raise NotImplementedEventType()
-    if category in ['payment_intent', 'payout', 'plan', 'product', 'recipient',
+
+    if category.startswith('payout'):
+        object_id = object_['id']
+        link = "https://dashboard.stripe.com/payout/{}".format(object_id)
+        amount = amount_string(object_["amount"], object_["currency"])
+        body_t = "**[Payout]({link})** for amount **{amount}** has {end}."
+
+        if event == "canceled":
+            end = "been canceled"
+            topic = "Payout {}".format(object_id)
+        if event == "created":
+            end = "been created"
+            topic = "Payout {}".format(object_id)
+        if event == "failed":
+            end = "failed"
+            topic = "Payout {}".format(object_id)
+        if event == "paid":
+            end = "been paid"
+            topic = "Payout {}".format(object_id)
+        if event == "updated":
+            end = "been updated"
+            topic = "Payout {}".format(object_id)
+
+        body = body_t.format(
+            description=object_['description'],
+            id=object_id,
+            link=link,
+            amount=amount,
+            end=end
```

Comment:
Are `description` and `id` even being used?

Reply:
Nope! I'll remove them.